Source/WebKit/NetworkProcess/PreconnectTask.cpp

```diff
@@ -82,8 +82,8 @@ void PreconnectTask::willSendRedirectedRequest(ResourceRequest&&, ResourceReques
 
 void PreconnectTask::didReceiveResponse(ResourceResponse&& response, PrivateRelayed, ResponseCompletionHandler&& completionHandler)
 {
-    ASSERT_NOT_REACHED();
```

Comment:
This seems fine, but it seems specific to cURL.  On Cocoa platforms the preconnect task should indeed never receive a response.  Could you keep the ASSERT_NOT_REACHED() inside a #if PLATFORM(COCOA) or #if !USE(CURL)?

Reply:
Thanks. I have changed it in https://github.com/WebKit/WebKit/commit/e3442d872260cfae9378878e3e895e7fe36a9557